repos/spack_repo/builtin/packages/blast_plus/package.py

```diff
@@ -168,4 +171,10 @@ def configure_args(self):
         else:
             config_args.append("--without-python")
 
+        with when("@2.15:"):
+            config_args.append("--with-sqlite={0}".format(self.spec["sqlite"].prefix))
+
+        with when("@2.17:"):
+            config_args.append("--with-zstd={0}".format(self.spec["zstd"].prefix))
```

Comment:
Nit: You could use f-strings here, e.g., 


```suggestion
            config_args.append(f"--with-sqlite={self.spec["sqlite"].prefix}")

        with when("@2.17:"):
            config_args.append(f"--with-zstd={self.spec["zstd"].prefix}")
```

Reply:
@tldahlgren I considered that, though all of the other strings in the package used `.format()`, so I followed that pattern. If you'd prefer, I could change all of the formatted strings to f-strings.

Reply:
Makes sense. You can leave it.

I'll give the maintainer some time to comment before merging.

Reply:
Sorry, I already changed it :D

Reply:
🤦 Thanks for catching the double-to-single quote correction in my suggestion.